src/tsickle_compiler_host.ts

```diff
@@ -318,36 +319,20 @@ export class TsickleCompilerHost implements ts.CompilerHost {
   }
 
   // Delegate everything else to the original compiler host.
-
-  fileExists(fileName: string): boolean {
-    return this.delegate.fileExists(fileName);
-  }
-
-  getCurrentDirectory(): string {
-    return this.delegate.getCurrentDirectory();
-  };
-
-  useCaseSensitiveFileNames(): boolean {
-    return this.delegate.useCaseSensitiveFileNames();
-  }
-
-  getNewLine(): string {
-    return this.delegate.getNewLine();
-  }
-
-  getDirectories(path: string) {
-    return this.delegate.getDirectories(path);
-  }
-
-  readFile(fileName: string): string {
-    return this.delegate.readFile(fileName);
-  }
-
-  getDefaultLibFileName(options: ts.CompilerOptions): string {
-    return this.delegate.getDefaultLibFileName(options);
-  }
-
-  getCanonicalFileName(fileName: string): string {
-    return this.delegate.getCanonicalFileName(fileName);
-  }
+  fileExists = this.delegate.fileExists;
```

Comment:
It would be neat if this works!  But I'm not sure -- does this.delegate get set up first, or do these run first?